docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,189 @@
+  following page:
 
-## Step 3/5 Install Teleport Agent on Server
+  https://goteleport.com/signup/
 
-Paste and run the script on a Linux Server:
+- Docker installed on your workstation. We will use a Docker container to
+  register a server with Teleport via the Teleport SSH Service. You can find
+  installation instructions on [Docker's
+  website](https://docs.docker.com/get-docker/).
 
-![Past Install Script](../../../img/cloud/runscript-3.png)
+  <Notice type="tip">
 
-Teleport Agent Installed
-![Completed Install](../../../img/cloud/completed-script-4.png)
+  Docker is only required for a local demo environment. If you want to get
+  started registering your servers in your infrastructure with Teleport, view
+  our documentation on how to [register servers and configure access to
+  them](../../server-access/introduction.mdx).
 
-## Step 4/5 Access Server
+  </Notice>
 
-Click **CLOSE**.  The server can now be accessed via SSH:
-![Access Server](../../../img/cloud/online-session-5.png)
+- The `tsh` client tool. Visit our [Installation Guide](../../installation.mdx)
+  for instructions. If you want to get started quickly, you can skip this for
+  now and access Teleport through your browser.
 
-Interact with your SSH session:
-![Open SSH](../../../img/cloud/online-ssh-5.png)
+## Step 1/4. Spin up your server
 
-Exit the session with `exit`. You can replay the session within Session Recordings:
-![Replay Session](../../../img/cloud/online-replay-5.png)
+Spin up a server on your workstation. In Step 2, we will register this server
+with your Teleport Enterprise Cloud tenant:
 
-## Step 5/5 Access from Command Line
+```code
+$ docker run -it ubuntu:22.10 /bin/bash
+```
+
+Make sure you have installed `curl` in your container:
+
+```code
+$ apt-get update
+$ apt-get install -y curl
+```
+
+Keep this shell open in your terminal for the next step.
+
+## Step 2/4. Install the Teleport SSH Service
+
+To install the Teleport SSH Service on your server, visit the Teleport Web UI at
+the address of your Teleport Enterprise Cloud tenant. For example, visit:
+
+https://mytenant.teleport.sh
+
+If this is your first time visiting the Web UI, you will see a prompt to add
+your first resource: 
+
+![Add your first
+resource](../../../img/cloud/getting-started/add-my-first-resource.png)
+
+This will take you to the Teleport Discover UI:
+
+![Select resource
+type](../../../img/cloud/getting-started/select-resource-type.png)
+
+Leave **Server** selected and click **Next**.
+
+You will see a script that you can copy from the Teleport Web UI:
+
+![Configure resource](../../../img/cloud/getting-started/configure-resource.png)
+
+Copy the script and paste it into the shell you opened in the previous step.
+Remove `sudo` from the start of the script so you can run it on your container.
+It should look something like this:
+
+```text
+$ bash -c "$(curl -fsSL https://TENANT_SUBDOMAIN.teleport.sh/scripts/SCRIPT_NAME/install-node.sh)" 
```

Comment:
This is why I used the `text` label—is there an alternative that users will be less likely to think is a copy-pastable command?

Reply:
My bad, I assumed it was `code` because of the `$` prompt. Possible improvements:
- Get rid of the `$` prompt
- Change it to a quote block with `>`
- Make it part of the paragraph above using inline code tics.

Reply:
Addressed in 7ae87d817481cde63103bc06f6e7780f82345413